### PR TITLE
Revalidate project page every 10s

### PR DESCRIPTION
**Pull request by tomquirk**
I think thisll work

### CODE REVIEW

File: src/utils/server/pages/props.ts [hunk 1]
```diff
@@ -66,7 +69,14 @@ export async function getProjectProps(
...
     const cv = isV3ProjectResult ? CV_V3 : CV_V2
 
-    return { props: { metadata, projectId, cv } }
+    return {
+      props: {
+        metadata,
+        projectId,
+        cv,
+      },
+      revalidate: 10, // 10 seconds https://nextjs.org/docs/api-reference/data-fetching/get-static-props#revalidate
```
Comment (tomquirk) [inline src/utils/server/pages/props.ts:78]: This is the only change worth caring about in this PR

File: src/components/modals/IssueErc20TokenModal.tsx [hunk 2]
```diff
@@ -38,12 +37,10 @@ const useIssueErc20TokenTx = ():
 export function IssueErc20TokenModal({
   visible,
   onClose,
-  isNewDeploy,
   onConfirmed,
 }: {
   visible: boolean
   onClose: VoidFunction
-  isNewDeploy?: boolean
```
Comment (tomquirk) [inline src/components/modals/IssueErc20TokenModal.tsx:46]: unused

File: src/pages/p/[handle].page.tsx [hunk 3]
```diff
@@ -122,17 +124,11 @@ function V1Dashboard() {
...
 
-  // Checks URL to see if user was just directed from project deploy
   const handle = router.query.handle as string
-  const isNewDeploy = Boolean(router.query.newDeploy)
 
   if (!handle) return <Project404 projectId={handle} />
   if (!projectId) return <Loading />
   if (projectId === 0) {
-    if (isNewDeploy) {
-      return <NewDeployNotAvailable handleOrId={handle} />
```
Comment (tomquirk) [inline src/pages/p/[handle].page.tsx:133]: no v1 create no more

File: src/pages/v2/p/[projectId]/components/V2V3Dashboard.tsx [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
 import { V2V3Project } from 'components/v2v3/V2V3Project'
 import { layouts } from 'constants/styles/layouts'
+import { ProjectMetadataContext } from 'contexts/projectMetadataContext'
 import { TransactionProvider } from 'providers/TransactionProvider'
 import { NftRewardsProvider } from 'providers/v2v3/NftRewardsProvider'
 import { VeNftProvider } from 'providers/v2v3/VeNftProvider'
+import { useContext } from 'react'
+
+export function V2V3Dashboard() {
+  const { projectId } = useContext(ProjectMetadataContext)
```
Comment (tomquirk) [inline src/pages/v2/p/[projectId]/components/V2V3Dashboard.tsx:11]: context > prop

Comment (wraeth-eth) [reply src/pages/v2/p/[projectId]/components/V2V3Dashboard.tsx:11]: back it

File: src/utils/server/pages/props.ts (hunk 1 above)
Comment (wraeth-eth) [reply src/utils/server/pages/props.ts:78]: only thing I worry about this is blowing up our vercel edge cost

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**